docs/readmes/contributing/contribute_onboarding.md

```diff
@@ -0,0 +1,122 @@
+        title="✅" ; sound="ping"
+    else
+        title="❌" ; sound="basso"
+    fi
+    terminal-notifier -title "$title" -message "Finished at $(date +%X)" -sound "$sound"
+}
+```
+
+3. Install the [Homebrew package manager](https://brew.sh/)
+4. Install [terminal-notifier](https://github.com/julienXX/terminal-notifier) via `brew install terminal-notifier`
+5. Install [IntelliJ Ultimate Edition](https://www.jetbrains.com/idea/) via `brew install --cask intellij-idea`
+6. [Import](https://www.jetbrains.com/help/idea/sharing-your-ide-settings.html#import-export-settings) @hcgatewood's IntelliJ settings: [intellij-hcgatewood.zip](https://www.dropbox.com/s/2i38wrfrfhjyicz/intellij-hcgatewood.zip?dl=0)
+7. Clone the [Magma repository](https://github.com/magma/magma) via `git clone git@github.com:magma/magma.git ${MAGMA_ROOT}`
+
+### Install Magma locally
+
+Install Magma locally and get everything running.
+
+**Howto**
+
+1. Follow the [prerequisites guide](https://magma.github.io/magma/docs/next/basics/prerequisites) and install all development tools, up to but not including the "Build/Deploy Tooling" section
+2. Run all Orc8r tests
+    1. Via Docker build script: `cd ${MAGMA_ROOT}/Orc8r/cloud/docker && ./build.py -t ; noti`
+    2. [Via IntelliJ](https://magma.github.io/magma/docs/next/Orc8r/development/testing_tips)
+3. Follow the [quick start guide](https://magma.github.io/magma/docs/next/basics/quick_start_guide) to get an AGW and Orc8r instance running on your dev machine
+4. Visit the local [Swagger UI](https://swagger.io/tools/swagger-ui/) view of our REST API (URL is in @hcgatewood's Google Chrome bookmarks) and [list the set of managed networks](https://localhost:9443/apidocs/v1/#/Networks/get_networks) -- there should be one named "test"
+    - You will need to toggle a Google Chrome preference to [allow insecure localhost](https://superuser.com/questions/772762/how-can-i-disable-security-checks-for-localhost)
+
+Note: remember to periodically call `docker system prune` to clear outdated Docker artifacts from your dev machine.
+
+### Complete a starter task
+
+If you haven't already, join our community Slack channel via the link from the [Community](https://www.magmacore.org/community/) page and say hello in `#general`. We can point you toward a good starter task. You can also use the [`good first issue` tag on our GitHub repo to search for good starter tasks](https://github.com/magma/magma/labels/good%20first%20issue).
+
+As you're working on your starter task, refer to [Development Workflow](./contribute_workflow.md) for guidance on how to author a pull request and [Code Conventions](./contribute_conventions.md) for Magma conventions. Once your pull request is approved and merged, you're now an official contributor to the Magma project!
+
+### Next steps
+
+The community Slack channel is a great place to get connected. From there, we can coordinate choosing appropriate tasks given your skill set and the Magma roadmap.
```

Comment:
'community Slack channel' sounds a bit confusing as technically there is no 'community' channel in the Magma workspace. I would either say 'community Slack' or specify a channel directly.